.gitignore

```diff
@@ -18,6 +18,10 @@
 /Library/PinnedKegs
 /Library/PinnedTaps
 /Library/Taps
+/Library/Vendor/Ruby/Versions/*
+
+# Unignore our vendor symlink.
+!/Library/Vendor/Ruby/Versions/Current
```

Comment:
We will always use symlink to find the vendor tool. As result, to update it(e.g. OpenSSL Security update) is as simple as updating the symlink.